CHANGES.rst

```diff
@@ -50,6 +50,9 @@ Features
 Other changes
 +++++++++++++
 
+- Exceptions during publishing are now re-raised in a new exceptions debug mode
```

Comment:
```suggestion
- Exceptions during publishing are now re-raised in a new exceptions debug mode,
  see also `documentation <https://zope.readthedocs.io/en/latest/wsgi.html#werkzeug>`_.
```

Reply:
Yes, good idea. I added the link to the documentation.